Makefile.toml

```diff
@@ -119,6 +119,14 @@ AMI_DATA_FILE_SUFFIX = "amis.json"
 # The type of testsys test that should be run.
```

Comment:
Please follow the convention suggested in a previous PR about the commit messages format